docs/en/programming.md

```diff
@@ -420,3 +420,17 @@ Subscribe to the updates (threads and submission) from a paritcular Hacker News
 ### weekly
 
 <RouteEn author="Pulset" example="/zooTeam/weekly" path="/zooTeam/weekly" radar="1" rssbud="1"/>
+
+## ui.dev
+
+### BYTES - Your weekly dose of JS
+
+Staying informed on the JavaScript ecosystem has never been so entertaining. Delivered every Monday and Thursday, for free.
+
+<RouteEn author="meixger" example="/uidev/bytes" path="/uidev/bytes"/>
+
+### React Newsletter
+
+Stay up to date on the latest React news, tutorials, resources, and more. Delivered every Tuesday, for free.
+
+<RouteEn author="meixger" example="/uidev/reactnewsletter" path="/uidev/bytes"/>
```

Comment:
Please use the domain name for namespaces, i.e., `/bytes/something` for feeds from `bytes.dev` instead of `/uidev/bytes/something` and `/reactnewsletter/something` for feeds from `reactnewsletter.com`